composer.json

```diff
@@ -27,7 +27,7 @@
         }
     },
     "require": {
-        "php": "^7.1",
+        "php": "^7.2 || ~8.0.0",
```

Comment:
We can, and should, bump this to 7.3, as 7.2 goes out of support in 11 days.

Reply:
Actually, its PHP 7.3 which goes out of support in 11 days.
https://www.php.net/supported-versions

Reply:
> Actually, its PHP 7.3 which goes out of support in 11 days.

True, but there's no need to drop support for PHP 7.3 just yet.

Reply:
> True, but there's no need to drop support for PHP 7.3 just yet.

Yup, just wanted to fix that message as it states that PHP 7.3 might be supported for another year which aint true.

Reply:
@boesing Sorry, let me clarify: PHP 7.2 goes end-of-life in 11 days. PHP 7.3 continues to receive security patches for another year. Our security window is for versions that receive updates of any kind, so it's those getting both active support and those getting security patches. This is why I suggested bumping the minimum to 7.3.